topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.md

```diff
@@ -0,0 +1,514 @@
+```
+
+## Visualize assemblies using Bandage
+
+To get a sense of how well our data assembled, and to determine whether the contigs are chomosomal or plasmid DNA (the former being linear sequences while plasmids are circular molecules), **Bandage** {% icon tool %} can give a clear view of the assembly.
```

Comment:
```suggestion
To get a sense of how well our data was assembled, and to determine whether the contigs are chomosomal or plasmid DNA (the former being linear sequences while plasmids are circular molecules), **Bandage** {% icon tool %} can give a clear view of the assembly.
```

feel like original version was fine too but,dunno, reads slightly like it self assembled? Felt slightly off